src/routers/follower.js

```diff
@@ -0,0 +1,7 @@
+// import { Router } from 'express';
```

Comment:
@Raymond-Osy remember to remove this code if they are not needed

Reply:
Sure, thanks